src/crewai/cli/tools/api.py

```diff
@@ -11,14 +11,14 @@ def get(self, handle: str):
     def publish(
         self,
         handle: str,
-        public: bool,
+        is_public: bool,
```

Comment:
pyright complained about this being a reserved word (?), so I changed.

Reply:
How I wish python had explicit public, private, and other visibility keywords.